botocore/docs/shape.py

```diff
@@ -83,6 +83,17 @@ def traverse_and_document_shape(self, section, shape, history,
                     section=section)
             history.pop()
 
+    def _append_special_param_docstring(self, shape, section):
+        special_docs = {
+            'idempotencyToken':
+            'Please note this parameter is automatically populated '
```

Comment:
This formatting is confusing, and I'm a bit surprised it works. Initially I thought this was a list.